components/button/index.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types'
+import styled, { css } from 'styled-components'
+
+const StyledButton = styled.button`
+  background-color: ${(props) => props.theme.colors[props.type]};
+
+  ${(props) => props.type === 'primary' && css`
```

Comment:
White should be the default color. I don't think we should check for `props.type === 'primary'`. Otherwise other button types won't have a default color and we will have to define the color for each of them.